src/mimeparser.rs

```diff
@@ -2123,6 +2137,9 @@ fn get_mime_type(
             mime::OCTET_STREAM => match filename {
                 Some(filename) => {
                     match message::guess_msgtype_from_path_suffix(Path::new(&filename)) {
```

Comment:
Probably it's better not to call `guess_msgtype_from_path_suffix()` for chat messages at all, but keep `File` if the sender didn't specify particular mimetype